src/components/PoweredByText.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { translateRaw } from '@translations';
+
+import nansenLogo from '@assets/images/credits/credits-nansen.svg';
+import enslogo from '@assets/images/ens/ensIcon.svg';
+import coinGeckoIcon from '@assets/images/credits/credits-coingecko.png';
+import zapperLogo from '@assets/images/defizap/zapperLogo.svg';
+import { SPACING } from '@theme';
+import { Typography } from '@components';
+
+const Wrapper = styled.div<{ css?: string }>`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  ${({ css }) =>
+    css &&
+    `
+      ${css}
+    `}
+`;
+
+const Logo = styled.img`
+  height: 24px;
+`;
+
+const Text = styled(Typography)<{ flipped?: boolean }>`
+  ${({ flipped }) =>
+    flipped &&
+    `
+margin-left: ${SPACING.XS};
+`}
+  ${({ flipped }) =>
+    !flipped &&
+    `
+margin-right: ${SPACING.XS};
+`}
+`;
+
+export enum PoweredByProvider {
+  NANSEN,
+  ENS,
+  COINGECKO,
+  ZAPPER
```

Comment:
Prefer named `enums`.
Otherwise they are given numbers by TS and become painful to debug.
```
enum PoweredByProvider {
  NANSEN = 'NANSEN'
  //...
}
```
Since it's a config object, we can actually do it the other way around and make more succinct:
```
interface PoweredByProvider = { 
  icon: any;
  text: string;
  flipped?: boolean;
  fontSize?: string;
  css?: string 
}

const providers: Record<string, PoweredByProvider> = {
  NANSEN: {
    icon: nansenLogo,
    text: translateRaw('POWERED_BY'),
    fontSize: '12px',
    css: `
      position: absolute;
      bottom: ${SPACING.BASE};
    `
  },
}

// Derive the names from the config object itself
export type PoweredByProviders = key in providers

```